backend/tests/test_query_count.py

```
@@ -0,0 +1,77 @@
+        try:
+            count, response = self._count_queries(
+                self.client.get,
+                reverse('dashboard')
+            )
+            
+            # Should be less than 20 queries
+            print(f"Dashboard queries: {count}")
+            self.assertLess(count, 20, 
+                f"Dashboard made {count} queries, expected < 20")
+            self.assertEqual(response.status_code, 200)
+        except:
```

Ruff (0.15.21)

[error] Do not use bare `except`

(E722)